pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -116,7 +123,35 @@ static int _event(struct dtls_context_t *ctx, session_t *session,
             break;
     }
 #endif  /* ENABLE_DEBUG */
-    mbox_put(&sock->mbox, &msg);
+    if (code != DTLS_EVENT_CONNECT) {
+        mbox_put(&sock->mbox, &msg);
+    }
+#ifdef SOCK_HAS_ASYNC
+    if (sock->async_cb != NULL) {
+        switch (code) {
+            case DTLS_ALERT_CLOSE_NOTIFY:
```

Comment:
When will this event occur?

Reply:
@pokgak? :-)

Reply:
This event will occur when the other side of the session calls `sock_dtls_session_destroy()` e.g. client calls `session_destroy()`, then the server receive this event.

Reply:
Let's add a comment to aid people reading that code in the future ;)